### PR TITLE
Tickets/dm 41005

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: bin/run_mtmount_ccw_only
Comment (tribeiro) [inline bin/run_mtmount_ccw_only:1]: I think this file is no longer needed right?

**Review by tribeiro — APPROVED** [review]
Nice! thanks you so much!

File: bin/run_mtmount_ccw_only
Comment (wvreeven) [reply bin/run_mtmount_ccw_only:1]: It's not except for my local tests. I've removed it.

Comment (wvreeven) [reply bin/run_mtmount_ccw_only:1]: It is required by `test_bin_script`. I removed the script and that test.

**Pull request merged**